### PR TITLE
feat: add my blog series

### LINKED ISSUE #529: [Resource] <Chrissy’s Open Source Survival Guide>
### Site URL

https://chrissycodes.hashnode.dev/

### Description of the above provided site

Whether you are a coding newbie or an experienced developer looking to keep your skills sharp, my series will help you navigate the fun and whimsical world of open source projects.

### Category

Resources

### Sub Category

open source

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
Fixes #529 

## Changes proposed

Added my blog series to the open source category 

## Note to reviewers

Feedback would be appreciated

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Hey Chrissy, it's not showing it on the site,

**Comment by CBID2** [comment]
> Hey Chrissy, it's not showing it on the site,

I noticed that too @rupali-codes. Did I put in the right categories?

File: database/opensource/opensource_blogs.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 [
-    
-]
\ No newline at end of file
+   {
+      "name": "Chrissy’s Open Source Survival Guide",
+      "description": "Whether you are a coding newbie or an experienced developer looking to keep your skills sharp, my series will help you navigate the fun and whimsical world of open source projects.",
+      "url": "https://chrissycodes.hashnode.dev/",
+      "category": "Resources",
```
Comment (rupali-codes) [inline database/opensource/opensource_blogs.json:6]:
```suggestion
      "category": "opensource",
```

File: database/opensource/opensource_blogs.json [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+      "subcategory": "Open Source"
```
Comment (rupali-codes) [inline database/opensource/opensource_blogs.json:7]:
```suggestion
      "subcategory": "opensource_blogs"
```

Comment (rupali-codes) [inline database/opensource/opensource_blogs.json:7]:
```suggestion
      "subcategory": "opensource_blogs"
```

**Comment by rupali-codes** [comment]
> > Hey Chrissy, it's not showing it on the site,
> 
> I noticed that too @rupali-codes. Did I use the right categories?

Yeah, the category and subcategory were incorrect

**Comment by CBID2** [comment]
> > > Hey Chrissy, it's not showing it on the site,
> 
> > 
> 
> > I noticed that too @rupali-codes. Did I use the right categories?
> 
> 
> 
> Yeah, the category and subcategory were incorrect

Ohh ok. What should I put to correct it @rupali-codes?

**Review by CBID2 — COMMENTED** [review]
LGTM

**Review by ujjawaltyagii — APPROVED** [review]
@CBID2  Looks good to me !! 👍 (I really enjoyed reading your blog post "5 Important Lessons I Learned From Creating An Online Community")

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**